msal/src/internal/java/com/microsoft/identity/client/MsalUtils.java

```diff
@@ -300,10 +300,8 @@ static String getChromePackage(final Context context) {
                 if (!MsalUtils.isEmpty(key) && !MsalUtils.isEmpty(value)) {
                     decodedUrlMap.put(key, value);
                 }
-                //CHECKSTYLE:OFF: checkstyle:EmptyBlock
             } catch (final UnsupportedEncodingException e) {
-                //CHECKSTYLE:ON: checkstyle:EmptyBlock
-                // TODO: log here.
+                Logger.error(MsalUtils.class.getSimpleName(), null, "URL form decode failed.", e);
```

Comment:
Add TAG in MsalUtils class and use it instead.

Reply:
Fixed 👍